app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,31 @@
+module SessionsHelper
+  def sign_in(user)
+    token = generate_token
+    user.update_attribute(:remember_token, token)
+    cookies.signed.permanent[:remember_token] = token
+    set_current_user(user)
+  end
+
+  def sign_in?
```

Comment:
this method should also be defined as a helper method so you can use it on your views

Reply:
Both methods `sign_in?` and `set_current_user(user)` are already helper methods